cmd/operator/main.go

```diff
@@ -239,7 +240,7 @@ func main() {
 	if opcfg.GetLoggingFilePath() != "" {
 		log.Printf("Now logging in file %s", opcfg.GetLoggingFilePath())
 	}
-
+	var multibroadcaster = record.NewBroadcasterWithCorrelatorOptions(record.CorrelatorOptions{BurstSize: 100})
```

Comment:
Let's just make this configurable through an env var. Look at examples in opcfg/config.go5(DEPLOY_WITH? WEBHOOKS_ENABLED). See how and where they are changed and do the same for the burstsize. By default it will be 25(k8s default), if a value lower than 25 is passed we still pick 25.

Reply:
Made the change. Please take a look.

Reply:
@LiboYu2,  I do not see the change.

Reply:
Forgot to push it. My bad. Now you should be able to see it.